### ISSUE TITLE
Dropdown: Accessibility : On Expanding dropdown it should Narrate Expand/Collapse

### ISSUE THREAD

**Issue by ama00527** [primefaces/primereact#7415]
### Describe the bug

While using DropDown the Expand/Collapse should be Narrated so that it can help user to understand the the dropdown is expanded with the list.

### System Information

```Shell
primereact 10.8.3
```

### Steps to reproduce the behavior

1. Use NVDA speech log
2. Make it focus on the dropdown
3. Press Enter.

### Expected behavior

Expected behaviour: Screen reader should narrate the value as expanded/collapsed
